packages/graphiql/cypress/integration/init.spec.js

```diff
@@ -0,0 +1,53 @@
+
+const testQuery = `{
+longDescriptionType {
+  id
+  image
+  hasArgs
+  test {
+    id
+    isTest
+    __typename
+  }
+ }
+}`
+
+const mockSuccess = {
+  "data": {
+    "longDescriptionType": {
+      "id": "abc123",
+      "image": "/images/logo.svg",
+      "hasArgs": "{\"defaultValue\":\"test default value\"}",
+      "test": {
+        "id": "abc123",
+        "isTest": true,
+        "__typename": "Test"
+      }
+    }
+  }
+}
+
+
+describe('GraphiQL On Initialization', function() {
+  it('Renders without error', function() {
+    cy.visit(`/?query=${testQuery}`)
```

Comment:
Every cypress test should take the form (setup) > action > assertion. [Ref](https://docs.cypress.io/guides/getting-started/writing-your-first-test.html#Write-a-real-test)

There are circumstances where this `visit` will throw errors, but I don't think it's really a sufficient assertion - we should add something like checking the query pane is visible or similar that requires that the page is of the shape we accept as opposed to an error page or a blank page.